scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -573,7 +559,8 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     def unapply(token: Token): Boolean = {
       token.is[Ident] || token.is[KwSuper] || token.is[KwThis] ||
       token.is[LeftParen] || token.is[At] || token.is[Underscore] ||
-      token.is[Unquote] || (token.is[Literal] && dialect.allowLiteralTypes)
+      token.is[Unquote] || (token.is[Literal] && dialect.allowLiteralTypes) ||
+      token.is[MacroSplicedIdent]
```

Comment:
Shouldn't we also have `token.is[MacroQuotedIdent] ` here ?

Reply:
correct, fixed